src/components/packages/Package.js

```diff
@@ -304,7 +313,7 @@ class Package extends Component {
                                         <span><strong>{feature.value}</strong></span>
                                         <span className="fs-feature-title">{feature.title}</span>
                                     </span>
-                                    {Helper.isNonEmptyString(feature.description) && <Tooltip><Fragment>{feature.description}</Fragment></Tooltip>}
+                                    {Helper.isNonEmptyString(feature.description) && <Tooltip direction={this.getTooltipDirection()}><Fragment>{feature.description}</Fragment></Tooltip>}
```

Comment:
### 1) IMHO we don't need the `<Fragment>` here. 

`<Fragment>` is used to nest multiple JSX elements when needed. Please consider this example.

```jsx
const renderSomething = useRenderSomething();

return renderSomething ? (
  <ComponentOne />
  <ComponentTwo />
) : (
  <ComponentThree />
);
```

The example above will fail to compile because JSX compilers expect one root level element.

```jsx
return renderSomething ? (
  <Fragment>
    <ComponentOne />
    <ComponentTwo />
  </Fragment>
) : (
  <ComponentThree />
);
```

Now it will get compiled to

```js
 return renderSomething
    ? React.createElement(
        Fragment,
        null,
        React.createElement(ComponentOne, null),
        React.createElement(ComponentTwo, null)
      )
    : React.createElement(ComponentThree, null);
```

Since in our code `{feature.description}` has a root component (`<Tooltip>`), we don't need to add the fragment here. Essentially the code

```jsx
  return (
    <Tooltip direction={this.getTooltipDirection()}>
      {feature.description}
    </Tooltip>
  );
```

will be compiled to

```js
  return React.createElement(
    Tooltip,
    {
      direction: this.getTooltipDirection(),
    },
    feature.description
  );
```

I hope you also understand the significance of having a deeper mental model of JSX compiler through our discussion 😃 .

#### 2) Please refactor the `&&` short-circuit to ternary

For the same reasons mentioned above.